src/index.js

```diff
@@ -206,15 +225,13 @@ export function distinctState() {
 // ifAction helper
 export function ifAction(rawActions) {
   const actions = parseActions(rawActions);
-  return (action) => action.type === '@@redux/INIT' || action.type === '@@INIT'
-    || actions.indexOf(action.type) > -1;
+  return (action) => actions.indexOf(action.type) >= 0;
 }
 // /ifAction
 
 // excludeAction helper
 export function excludeAction(rawActions = []) {
   const actions = parseActions(rawActions);
-  return (action) => action.type === '@@redux/INIT' || action.type === '@@INIT'
-    || !(actions.indexOf(action.type) > -1);
+  return (action) => !(actions.indexOf(action.type) < 0);
```

Comment:
Ah, you are right, sorry about that :) I just don't like much comparing to weird `-1` value :)